skills/supervibe/references/stages.md

```
@@ -0,0 +1,537 @@
+<tr><td><strong>操作人</strong></td><td>{user}</td></tr>
+<tr><td><strong>时间</strong></td><td>{timestamp}</td></tr>
+</table>
+
+<h3>下一步</h3>
+<p>进行需求分析和技术设计。</p>'
+
+   curl -X POST "https://dev.azure.com/{org}/{project}/_apis/wit/workItems/{id}/comments?api-version=7.1-preview.3" \
+     -H "Authorization: Bearer $TOKEN" \
+     -H "Content-Type: application/json" \
+     -d "{\"text\": \"$COMMENT\"}"
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)